recipe/build.sh

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./CoinUtils
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
+
 set -e
 
-UNAME="$(uname)"
-export CFLAGS="${CFLAGS} -O3"
-export CXXFLAGS="${CXXFLAGS} -O3"
-export CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++11}"
+if [[ "${target_platform}" == win-* ]]; then
+  EXTRA_FLAGS="--enable-msvc"
+else
+  # Get an updated config.sub and config.guess (for mac arm and lnx aarch64)
+  cp $BUILD_PREFIX/share/gnuconfig/config.* ./CoinUtils 
+  cp $BUILD_PREFIX/share/gnuconfig/config.* .
+
+  export CFLAGS="${CFLAGS} -O3" # avoid test failures
+  export CXXFLAGS="${CXXFLAGS} -O3" # avoid test failures
+  export CXXFLAGS="${CXXFLAGS} -std=c++11" # macOS clang defaults to cxx17 but coin uses old register keyword
```

Comment:
I would suggest trying to remove `-O3`. I don't know of any reason why it's needed, tests are passing on all platforms on Github Actions without it. We shouldn't need `-std=c++11` either once we are on the latest release (2.11.11).

Reply:
@tkralphs apparently 2.11.11 is not available yet

Reply:
Yes, all versions within the same stable series are compatible, but in any case, when I make a release of Cbc, I usually make releases of all the dependencies as well. So the latest Cbc has been tested with the latest CoinUtils.

Reply:
Oops, sorry, the latest release is 2.11.10.

Reply:
It also seems like 2.11.10 is not ready for c17 yet. Could that be?

Reply:
There were some tries to remove the register keywords apparently but not all were removed until the master branch:

https://github.com/coin-or/CoinUtils/blob/640e1ccb670eb02269e3b6e87cd6b0eeacb3b671/CoinUtils/src/CoinOslFactorization2.cpp#L24

Reply:
Specifically, only one of two commits were ported to stable.

The following is missing:
https://github.com/coin-or/CoinUtils/commit/1700ed92c2bc1562aabe65dee3b4885bd5c87fb9